tools/report.collector.in

```diff
@@ -588,11 +588,12 @@ get_logfiles() {
     pattern="Starting Pacemaker"
 
     # - make sure we get something from the scheduler
-    pattern="$pattern\\|Calculated Transition"
+    pattern="$pattern\\|Calculated transition"
 
     # - cib and pacemaker-execd updates
     # (helpful on non-DC nodes and when cluster has been up for a long time)
     pattern="$pattern\\|cib_perform_op\\|process_lrm_event"
+    pattern="$pattern\\|Diff: +++ \\|Result of .* for .* on"
```

Comment:
Looking at this, I wondered why the previous logger test would not find the system logs. Then I noticed it only checks for the log facility in the corosync configration, not in the pacemaker sysconfig.

I think a better approach would be to fix that, so we don't have any chance of matching another application's messages with the above, which seem like they might be not specific enough. There is a @CONFIGDIR@ substitution variable to match /etc/sysconfig. We could call logger twice, once for the corosync logfacility, and once for PCMK_logfacility (if different), so we get all the logs if they're going to different files.

Reply:
I think we should remove the above line -- the new syslog checks should catch the logs without it, and we won't risk matching some other program that logs diffs. Otherwise I think this is ready to merge, thanks for all the help!

Reply:
I agree. removed it.